### PR TITLE
[Accuracy diff No.122] Fix accuracy diff for paddle.nn.functional.kl_div API

### PR DESCRIPTION
**Pull request by co63oc**
grad 精度已增加配置 
image
复制配置到 tester/api_config/5_accuracy/accuracy_gpu_error_grads_diff.txt

0-size 已修改 https://github.com/PaddlePaddle/Paddle/pull/73173

PaddleAPITest 测试通过
image

### CODE REVIEW

**Review by cangtianhuang — APPROVED** [review]
All backtests passed, LGTM!
